### PR TITLE
Admin page UI changes

### PR DESCRIPTION
**Pull request by ankkit24**
Added links for the below in the admin panel-
* Browser url
* Web console url
* Sling welcome page

- Also, the latest build date is displayed below the links

### CODE REVIEW

File: frontend/src/app/admin.tsx [hunk 1]
```diff
@@ -1,6 +1,11 @@
 import React, { Component } from 'react';
-import { Button } from '@patternfly/react-core';
 import { Redirect } from 'react-router-dom';
+import { Button } from '@patternfly/react-core';
+import { Grid, GridItem } from '@patternfly/react-core';
+import browseri from './sample.svg';
```
Comment (jmarrero) [inline frontend/src/app/admin.tsx]: You are importing these svgs (console1.svg, etc) but they are not being used, instead it looks like the SVG is copied on the admin.tsx as well. I would delete the paths with the svg vectors from this file and use the imported ones.

Comment (ankkit24) [reply frontend/src/app/admin.tsx]: Hey Joseph!
I have addressed the issue.
The images are appearing now.

**Comment by jmarrero** [comment]
@ankkit24 The code looks good now. However why jpg? and not svg like before? now when I make my window smaller it distorts the image.

**Comment by jmarrero** [comment]
It seems like using svg has a similar problem. I think this is fine for now.

**Pull request merged**